modules/homer/findpeaks/main.nf

```diff
@@ -0,0 +1,42 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+def VERSION = '4.11'
```

Comment:
I'm assuming you can't get the version from the tools somehow? That would be the safer option downstream when we automatically update this module.

Reply:
No, I wasn't able to find a way, this was what the `annotatePeaks` module was doing.